### ISSUE TITLE
Outdated docs for installation and possibly others.

### ISSUE THREAD

**Issue by wildwestrom** [fabd/kanji-koohii#209]
I'm going through `kanji-koohii/doc/Installation.md` and I'm suck on [Line 44](https://github.com/fabd/kanji-koohii/blob/master/doc/Installation.md#L44)

```console
$ npm run dev
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path /var/www/html/package.json
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, open '/var/www/html/package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-09-09T20_06_49_979Z-debug.log
```

What should I be doing instead to get things up and running?

**Comment by wildwestrom** [comment]
```md
You should see something like this (files are output to `src/web/build/pack/`):

                    Asset     Size          Chunks             Chunk Names
    landing-bundle.raw.js  678 KiB  landing-bundle  [emitted]  landing-bundle
    review-bundle.raw.js   580 KiB   review-bundle  [emitted]  review-bundle
    root-bundle.raw.js     731 KiB     root-bundle  [emitted]  root-bundle
    study-bundle.raw.js    497 KiB    study-bundle  [emitted]  study-bundle
```
I see no such thing. Maybe replace with this?
```
../web/build/dist/manifest.json                       1.60 KiB
../web/build/dist/assets/entry-landing.5a54af7b.js    0.08 KiB / brotli: 0.08 KiB
../web/build/dist/assets/entry-manage.d10910ae.js     3.03 KiB / brotli: 1.12 KiB
../web/build/dist/assets/entry-account.52d7ddf2.js    3.47 KiB / brotli: 1.34 KiB
../web/build/dist/assets/entry-review.a3ed3196.js     24.61 KiB / brotli: 7.12 KiB
../web/build/dist/assets/entry-common.40f6688f.js     39.36 KiB / brotli: 11.94 KiB
../web/build/dist/assets/entry-manage.409979e5.css    1.63 KiB / brotli: 0.57 KiB
../web/build/dist/assets/entry-study.ef7cf6ee.js      36.25 KiB / brotli: 9.34 KiB
../web/build/dist/assets/entry-review.930b9cf3.css    14.06 KiB / brotli: 3.34 KiB
../web/build/dist/assets/entry-study.1525e853.css     8.98 KiB / brotli: 2.33 KiB
../web/build/dist/assets/entry-landing.1d2d79b9.css   42.29 KiB / brotli: 7.02 KiB
../web/build/dist/assets/vendor.23149c00.js           73.11 KiB / brotli: 25.53 KiB
../web/build/dist/assets/entry-common.e15a5aaa.css    81.54 KiB / brotli: 15.14 KiB
```

**Comment by fabd** [comment]
Yes sorry about that I was postponing the update of the docs -- which is always a huge timesink for me -- until I push the changes to live.

I have quite a bit of updates already locally, I'll look at the changes you submitted in #210 and see if I can merge them, especially if I find it clearer (always interesting to see how someone else reasons about the same thing, and writing docs is hard >_>).

But yeah with the current `master` my routine is:

```
$ docker-compose exec web bash
```

Then from the php/apache container

```
$ cd vite/
$ vite
```

If I don't like the vite client lag (especially with Docker on Apple Silicon), the following I have already documented and will be merged soon:

change this line in coreWebResponse.php

```
const USE_DEV_SERVER = false;
```

then use

```
$ vite build --watch
```

This essentially renders the page with the CSS/JS built files just like production, 

just be aware that in this mode (`--watch`) tailwind JIT doesn't update unless you restart it, or figure out some workaround by using `nodemon` (I have no experience with that, just an idea I saw on a ViteJs repo issue thread).

There might be a better way to organize the folders to run Vite, I don't know.. the idea here is everything node related moved into the `src/vite/` subfolder.
